simphony/io/tests/test_h5_cuds.py

```diff
@@ -78,15 +178,72 @@ def test_incorrect_version(self):
 
 
 class TestParticlesCudsOperations(ParticlesCudsCheck, unittest.TestCase):
-    pass
+
+    def setUp(self):
+        ParticlesCudsCheck.setUp(self)
+        self.engines = []
+
+    def engine_factory(self):
+        filename = os.path.join(self.temp_dir, 'test.cuds')
```

Comment:
Once, https://github.com/simphony/simphony-common/pull/189/files#r34765387 is addressed, this needs to be looked at to ensure that proper cleanup of the temp file/directories occurs